airflow/jobs/scheduler_job.py

```diff
@@ -1358,8 +1358,8 @@ def check_trigger_timeouts(self, session: Session = None):
     def _find_zombies(self, session):
         """
         Find zombie task instances, which are tasks haven't heartbeated for too long
-        or have a no-longer-running LocalTaskJob, and send them off to the DAG processor
-        to be handled.
+        or have a no-longer-running LocalTaskJob, and register a TaskCallbackRequest,
```

Comment:
```suggestion
        or have a no-longer-running LocalTaskJob, and create a TaskCallbackRequest
```

A little better still?